lib/ObisNames.js

```diff
@@ -6,352 +6,352 @@
         130: "Voorspelling",
-        131: "Maximale limieten werkdag",
-        132: "Maximumgrenzen niet-werkdag",
-        133: "Minimale limieten werkdag",
-        134: "Minimumgrenzen niet-werkdag",
-        135: 'SLP maandag',
-        136: 'SLP Dinsdag',
-        137: 'SLP Woensdag',
-        138: 'SLP Donderdag',
-        139: 'SLP Vrijdag',
-        140: 'SLP zaterdag',
-        141: "SLP Werkdag",
-        142: "SLP-niet-werkdag",
+        131: "Maximale grenswaarden werkdag",
+        132: "Maximale grenswaarden rustag",
+        133: "Minimale grenswaarden werkdag",
+        134: "Minimale grenswaarden rustdag",
+        135: "SLP maandag",
+        136: "SLP dinsdag",
+        137: "SLP woensdag",
+        138: "SLP donderdag",
+        139: "SLP vrijdag",
+        140: "SLP zaterdag",
+        141: "SLP werkdag",
+        142: "SLP rustdag",
         143: "Geldbedrag",
-        144: "Voorspelde hoeveelheid geld",
-        145: 'Emmision CO2',
-        146: "Voorspelling Emmision CO2"
+        144: "Prognose geldbedrag",
+        145: 'CO2-uitstoot',
+        146: "Prognose CO2-uitstoot"
     }
 };
 
 const obisMeasurement = {
     0: { // Abstract Objects
         'en': {
-        	128: 'Temperature',
-            129: 'Status',
-        	130: 'Humidity',
-            131: 'Light Intensity',
-            132: 'Operating Hours',
-            133: 'Units',
-            96: 'Service entry',
-            97: 'Error message',
-            98: 'List',
-            99: 'Data profiles, Load profiles P.01/ P.02, Operation Log P.98/ P.99'
+            128: "Temperature",
+            129: "Status",
+            130: "Humidity",
+            131: "Light Intensity",
+            132: "Operating Hours",
+            133: "Units",
+            96: "Service entry",
+            97: "Error message",
+            98: "List",
+            99: "Data profiles, Load profiles P.01/ P.02, Operation Log P.98/ P.99"
         },
         'de': {
-        	128: 'Temperatur',
-            129: 'Status',
-        	130: 'Feuchte',
-            131: 'Lichtstärke',
-            132: 'Betriebsstunden',
-            133: 'Stückzahl',
-            96: 'Wartungseintrag',
-            97: 'Fehler',
-            98: 'Listen',
-            99: 'Datenprofile, Lastgang P.01/ P.02, Betriebslogbuch P.98/ P.99'
+            128: "Temperatur",
+            129: "Status",
+            130: "Feuchte",
+            131: "Lichtstärke",
+            132: "Betriebsstunden",
+            133: "Stückzahl",
+            96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen",
+            99: "Datenprofile, Lastgang P.01/ P.02, Betriebslogbuch P.98/ P.99"
         },
         "nl": {
-            128: "temperatuur",
+            128: "Temperatuur",
             129: "Status",
             130: "Vochtigheid",
             131: "Lichtintensiteit",
-            132: "Bedrijfsuren",
-            133: "Aantal stuks",
+            132: "In bedrijf",
```

Comment:
In bedrijf ? Volgens mij is dit de meeting hoeveel draai uren, in bedrijf kan alleen Ja/Nee zijn en hier komen aantallen uur